### PR TITLE
ansi: add EIEIO to strerror()

### PR DESCRIPTION
**Pull request by ElectrodeYT**

### CODE REVIEW

**Review by no92 — APPROVED** [review]
LGTM

File: options/ansi/generic/string-stubs.cpp [hunk 1]
```diff
@@ -455,6 +455,11 @@ char *strerror(int e) {
 	case ENOTUNIQ: s = "Name not unique on network (ENOTUNIQ)"; break;
 	case ERESTART: s = "Interrupted system call should be restarted (ERESTART)"; break;
 	case EUSERS: s = "Too many users (EUSERS)"; break;
+
+	#ifdef EIEIO
```
Comment (Dennisbonke) [inline options/ansi/generic/string-stubs.cpp:459]: Formatting issue, the #ifdef should not be indented here. Same for the #endif.

**Review by Dennisbonke — CHANGES_REQUESTED** [review]
Small formatting issue

File: options/ansi/generic/string-stubs.cpp (hunk 1 above)
Comment (ElectrodeYT) [reply options/ansi/generic/string-stubs.cpp]: Done

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**